11.0/base_requirements.txt

```diff
@@ -0,0 +1,67 @@
+# Odoo dependencies
+Babel==2.3.4
+MarkupSafe==0.23
+Python-Chart==1.39
+PyYAML==3.11
+Werkzeug==0.9.6
+argparse==1.2.1
+chardet==2.3.0
+colorama==0.3.2
+decorator==4.0.10
+docutils==0.12
+ebaysdk==2.1.5
+feedparser==5.2.1
+gevent==1.1.2
+gdata==2.0.18
+greenlet==0.4.10
+html2text==2016.9.19
+html5lib==0.999
+Jinja2==2.8
+lxml==3.5.0
+Mako==1.0.4
+mock==2.0.0
+num2words==0.5.4
+odfpy==1.3.4
+odoo-autodiscover==2.0.0
+ofxparse==0.16
+passlib==1.6.5
+Pillow==3.4.1
+psutil==4.3.1
+psycopg2==2.7.3.1
+psycogreen==1.0
+pyPdf==1.13
+pyinotify==0.9.4
+pyparsing==2.0.3
+pyserial==2.7
+python-dateutil==2.4.0
+pyldap==2.4.28
+python-openid==2.2.5
+python-stdnum==1.0
+pytz==2016.7
+pyusb==1.0.0b2
+qrcode==5.1
+reportlab==3.1.44
+requests==2.6.0
+roman==2.0.0
+simplejson==3.6.5
+six==1.10.0
+suds-jurko==0.6
+urllib3==1.9.1
+vatnumber==1.2
+vobject==0.8.2
+XlsxWriter==0.9.3
+xlwt==1.3.*
+xlrd==1.0.0
+# version in debian has problems
+pydot==1.2.3
+
+# Migration tools
+marabunta==0.7.2
+anthem==0.10.0.linux-x86_64
```

Comment:
I think we should always use last version here and we can pin it in proj requirement if needed

Reply:
Could be done but I'm more in favour of having it fixed here. So we know that this version of the odoo-project is shipped with a given version anthem.
What are the avantages in your solution?

Reply:
@simahawk I think we want it pinned because we have to generate a new image anyway. It will be easier to know what is is the image that way.

Reply:
@yvaucher you are right. Pardonne moi :)